### PR TITLE
16 as a frontend developer i need an about us page so users can see what the application is all about

### LINKED ISSUE #16: As a frontend developer, I need an About Us page, so users can see what the application is all about

### PR DESCRIPTION
**Pull request by bradhezh**
I have created an about page, with the unit test and selenium test.

### CODE REVIEW

File: groupassignment.side
Comment (JoshuaPeralta07) [inline groupassignment.side:1]: This selenium test file should not be in the project folder. We are using Selenium IDE extension in chrome this should be a separate file from the project files.

Comment (bradhezh) [reply groupassignment.side:1]: What do you mean by 'separate file'? Do you mean that the selenium test file should be in another folder?

Comment (JoshuaPeralta07) [reply groupassignment.side:1]: Yes, it should be outside the project folder. Run `npm test` on your branch to check if the test that you created for your component has passed

**Comment by JoshuaPeralta07** [comment]
[] - Have the requirements been met?
[] - Is the code easy to read?
[] - Do unit tests pass?
[] - Build pass?
[] - No unnecessary comments

**Comment by JoshuaPeralta07** [comment]
- [x] Have the requirements been met?
- [x] Is the code easy to read?
- [x] No unnecessary comments?
- [x] Is the code formatted correctly?
- [x] Do unit tests pass?
- [x] Does the system tests pass?
- [x] Build pass?
- [x] Code corrections made?

**Pull request merged**